components/PopupMenu.js

```diff
@@ -14,10 +14,10 @@ const Popup = styled(Box)`
   border-radius: 8px;
   background: white;
   box-shadow: 0px 4px 8px rgba(20, 20, 20, 0.16);
-  z-index: 1000;
+  z-index: ${props => props.zIndex || 1000};
```

Comment:
`0` is a valid z-index

```suggestion
  z-index: ${props => props.zIndex ?? 1000};
```